dev/io.openliberty.accesslists.internal/test/io/openliberty/accesslists/internal/test/UnitTestAccessListConfig.java

```diff
@@ -0,0 +1,385 @@
+package io.openliberty.accesslists.internal.test;
```

Comment:
Gordon we should add a copyright header to this new class.

Reply:
Apologies. Done.